src/main/resources/liquibase.properties

```diff
@@ -0,0 +1,4 @@
+url=jdbc:mysql://localhost:3306/db?serverTimeZone=UTC
+username=root
+password=root
+changeLogFile=src/main/resources/db/changelog/db.changelog-master.yaml
```

Comment:
do you really need a separate configuration file for liquibase setups?

Reply:
In the "Spring Data JPA" topic, specifically in the "Liquibase rollback" video,
 it is mentioned that if I need to perform a rollback,
 I should use a Liquibase configuration file and add the plugin in the pom.xml,
specifying the path to liquibase.properties.

Reply:
okay, but you don’t have rollback statements in your changesets